Scoop/Controller/Ride Posting/InitialPostRideViewController.swift

```diff
@@ -208,14 +208,14 @@ class InitialPostRideViewController: OnboardingViewController {
         arrivalLabel.text = "Arrival"
         
         departureLabel.snp.makeConstraints { make in
-            make.top.equalTo(departureTextField).offset(labelTop)
+            make.top.equalTo(departureTextField).inset(-labelTop)
```

Comment:
I originally also had this as inset of a negative number, but Reade commented in my previous PR and told me to change it to offset. This is quoted from his comment on my PR: 
"As a side note - I like to inset relative to the superview (because I know this view is contained inside the superview), but when I am positioning views relative to each other, I like to use offset"

Reply:
Feel free to disregard if you don't agree - there are a few more inset/offset comments later on as well but I couldn't tell if Reade was saying that it's standard practice or if it's personal preference

Reply:
Thats the weird thing tho, the offset with the positive number is different lol. Like the positioning is off when I change it. Also I think someone at some point told me to use insets and I've just been using them since lol